modules/openapi-generator/src/main/resources/r/api.mustache

```diff
@@ -515,7 +515,7 @@
       file_params["{{baseName}}"] <- httr::upload_file(`{{paramName}}`)
       {{/isHttr2}}
       {{#isHttr2}}
-      file_params["{{baseName}}"] <- curl::form_file(`{{paramName}}`)
+      file_params[["{{baseName}}"]] <- curl::form_file(`{{paramName}}`)
```

Comment:
minor suggestion: can you please update this line to use `{{{baseName}}}` instead of `{{baseName}}` ?

this will ensure moving forward there will be no surprise even if baseName contains special characters (that may be HTML escaped)